development-updates.md

```diff
@@ -57,6 +58,12 @@ your development updates in this document.
 - Add it to the [Table of Contents](#table-of-contents)
 - Keep the overall list of people sorted alphabetically.
 
+## Abdel Haifouli
+
+Development updates from [Abdel Haifouli](https://github.com/Ahaif)
+
+### [Update #1 - 2022.10.29](https://www.notion.so/EFP-update-1-7b6ddf0523964f94ba32a0311bf2cb24)
```

Comment:
Looks like I can't access the doc:
![image](https://user-images.githubusercontent.com/6136245/198849587-b053af83-1378-4111-af6d-17ad17e398d8.png)

It probably needs some tuning of permissions.

BTW, glad someone is using Notion, I like it. :)

Reply:
Yes my bad , thank you for the comment